app/src/main/java/com/mapbox/maps/testapp/examples/markersandcallouts/infowindow/InfoWindowActivity.kt

```diff
@@ -0,0 +1,108 @@
+package com.mapbox.maps.testapp.examples.markersandcallouts.infowindow
+
+import android.graphics.Bitmap
+import android.os.Bundle
+import androidx.appcompat.app.AppCompatActivity
+import com.mapbox.geojson.Point
+import com.mapbox.maps.MapView
+import com.mapbox.maps.Style
+import com.mapbox.maps.dsl.cameraOptions
+import com.mapbox.maps.plugin.annotation.generated.*
+import com.mapbox.maps.plugin.gestures.OnMapLongClickListener
+import com.mapbox.maps.plugin.gestures.addOnMapLongClickListener
+import com.mapbox.maps.plugin.gestures.removeOnMapLongClickListener
+import com.mapbox.maps.testapp.R
+import com.mapbox.maps.testapp.utils.BitmapUtils
+import java.text.DecimalFormat
+
+/**
+ * Test activity showcasing using the legacy InfoWindow API above Washington D.C
+ * using new View Annotation API.
+ *
+ * @see [legacy example code](https://github.com/mapbox/mapbox-gl-native-android/blob/main/MapboxGLAndroidSDKTestApp/src/main/java/com/mapbox/mapboxsdk/testapp/activity/infowindow/InfoWindowActivity.java)
```

Comment:
do we want to link to legacy codebase? we didn't reference any of the legacy activities in our codebase.

Reply:
Yeah, agree, will remove. Mentioning it in PR description should be enough (just to make it easier for us to navigate if needed)